.circleci/config.yml

```diff
@@ -185,22 +191,35 @@ jobs:
               exit 1
             fi
           name: Tag Release in GitHub
+      - run:
+          command: |
+            echo {} > package.json
+            version=$(jq -r .version composer.json)
+            ~/.yarn/bin/yarn add github-release-notes
+            ~/.yarn/bin/yarn gren release -m -t $version -T $GITHUB_ACCESS_TOKEN
+            ~/.yarn/bin/yarn gren release -o -t $version -T $GITHUB_ACCESS_TOKEN
```

Comment:
I would recommend running `yarn global add github-release-notes` which will create a symlink to it in (I think) /usr/local/bin that you can just run (like [this](https://github.com/ns8inc/protect-sdk-php/blob/master/.circleci/config.yml#L51)), so you don't need to do the `~/.yarn/bin/yarn` thing.

Reply:
The symlink seems to only exist after restarting the bash terminal when I tested which is why I'm running directly from the source